moduleframework/helpers/rpm_helper.py

```diff
@@ -104,14 +107,16 @@ def setRepositoriesAndWhatToInstall(self, repos=[], whattooinstall=None):
         if repos:
             self.repos = repos
         else:
-            self.repos += get_compose_url() or self.get_url()
+            self.repos += common.get_compose_url() or self.get_url()
```

Comment:
Here you use `common.`, but above only `print_info`. Any reason?
Keep the python style.

Reply:
I'll fix it. would be nice also to fix all these issues in next PR, we can do issue for this cleanup